src/main/java/core/basesyntax/service/impl/FruitServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.strategy.FruitStrategy;
+import core.basesyntax.strategy.OperationsStrategy;
+import java.util.List;
+
+public class FruitServiceImpl implements FruitService {
+    @Override
+    public void getAllOperationsStrategy(List<FruitTransaction> fruitFromCsvRow,
```

Comment:
the name of the method doesn't match to what it does
```suggestion
    public void processTransactions(List<FruitTransaction> fruitTransactions,
```
also rename  please variable `fruitFromCsvRow`, method should not know from which source data was delivered.